jaeger-core/src/main/java/io/jaegertracing/Configuration.java

```diff
@@ -332,26 +337,36 @@ public MetricsFactory getMetricsFactory() {
     private Number param;
 
     /**
-     * HTTP host:port of the sampling manager that can provide sampling strategy to this service.
+     * HTTP URL of the sampling manager that can provide sampling strategy to this service.
      * Optional.
      */
-    private String managerHostPort;
+    private String serverUrl;
 
     public SamplerConfiguration() {
     }
 
     public static SamplerConfiguration fromEnv() {
+      String configManagerEndpoint = getProperty(JAEGER_CONFIG_MANAGER_ENDPOINT);
+      if (configManagerEndpoint == null) {
+        if (getProperty(JAEGER_SAMPLER_MANAGER_HOST_PORT) != null) {
+          configManagerEndpoint = "http://" + getProperty(JAEGER_SAMPLER_MANAGER_HOST_PORT) + "/sampling";
```

Comment:
nit: `String.format()` might be cleaner here.

Reply:
If we go with HOST_PORT, shouldn't need to use any sort of string manipulation; pending confirmation that we're happy to diverge from the Go lib.